config.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"github.com/mitchellh/go-homedir"
```

Comment:
Ok, now that we're bringing in dependencies, should we use vending now? https://blog.gopheracademy.com/advent-2015/vendor-folder/